integration/spark/src/main/scala/org/apache/spark/sql/execution/command/partition/CarbonAlterTableDropHivePartitionCommand.scala

```diff
@@ -105,6 +106,25 @@ case class CarbonAlterTableDropHivePartitionCommand(
             ifExists,
             purge,
             retainData).run(sparkSession)
+
+          // move  the partition files to trash folder which are dropped
+          val droppedPartitionNames = partitions.map { partition =>
+            partition.spec.map { specs => specs._1 + "=" + specs._2}
+          }
+          val droppedPartitionName = droppedPartitionNames.head.mkString("/")
```

Comment:
i think, we should move the partition stale folders under respective trash partition folders. here, you are taking the first dropped partition folder to move other partitions data also.